solutions/instances/variables.tf

```diff
@@ -276,6 +276,35 @@ variable "enable_at_event_routing_to_cloud_logs" {
   default     = true
 }
 
+##############################################################################
+# Metric Routing Variables
+##############################################################################
+
+variable "metrics_router_routes" {
+  type = object({
+    name = string
+    rules = list(object({
+      action = string
+      targets = list(object({
+        id = string
+      }))
+      inclusion_filters = list(object({
+        operand  = string
+        operator = string
+        values   = list(string)
+      }))
+    }))
+  })
+  default     = null
+  description = "Routes for IBM Metrics Router"
+}
+
+variable "enable_metric_routing_to_cloud_monitoring" {
+  type        = bool
+  description = "Whether to enable metric routing from IBM Cloud Metric Routing to Cloud Monitoring."
```

Comment:
Metric -> Metrics

Also, you have used IBM Cloud Metrics routing / IBM Metrics router/Metrics routing --> lets be consistent in all the places. If I search, I get `IBM Cloud Metrics Routing`